### ISSUE TITLE
Child_added supported in sync? -> Adding to Documentation

### ISSUE THREAD

**Issue by ReactNativeBlog** [n6g7/redux-saga-firebase#84]
First of all: Great Work!

I'm currently wondering which data is transferred on a sync in case a child was added/remove/... in the realtime database?
Is the whole data/ref transferred or only the delta?

Many thanks for clarification.

**Comment by n6g7** [comment]
Thanks!

Yes, the [`database.sync`](https://n6g7.github.io/redux-saga-firebase/reference/database#sync) method currently uses the `value` event so it will send the whole data in the ref.

If you want to only receive new or deleted children you can choose the event you want to use (`child_added`, `child_removed `, `child_changed ` or `child_moved `, see the [`Reference.on` docs](https://firebase.google.com/docs/reference/js/firebase.database.Reference#on) for more details), in the [`database.channel`](https://n6g7.github.io/redux-saga-firebase/reference/database#channel) method:

```js
function * syncAddedChildren () {
  const channel = yield call(rsf.database.channel, 'items', 'child_added') // 👈 add a third argument here 😉

  while (true) {
    const data = yield take(channel)
    yield put(actionCreator(data))
  } catch (err) {
    yield put(failureActionCreator(err))
  } finally {
    if (yield cancelled()) channel.close()
  }
}
```

It never occurred to me that there was a strong use case for this but if that's the case for you we can definitely add an API to make this a lot simpler. :)
I'm thinking of something like:
```js
function * syncAddedChildren () {
  yield fork(
    rsf.database.sync,
    'items', 
    { successActionCreator: ..., failureActionCreator: ... },
    'child_added' // 👈 we could add a third argument here 🙂
  )
}
```

Please let me know if that would be interesting to you! :)

**Comment by ReactNativeBlog** [comment]
Great !!!

**Comment by n6g7** [comment]
Thanks for the feedback, this is much appreciated. :D

> Possible scenarios for such an API are
>- chat system
>- events with bad wifi and >10.000 app users

That sounds reasonable, I'll keep this issue open and notify you when this is ready. :)

**Comment by n6g7** [comment]
@ReactNativeBlog I've just released [v0.8.0](https://github.com/n6g7/redux-saga-firebase/releases/tag/v0.8.0) with this change!
Run `yarn upgrade redux-saga-firebase` and you should be able to use the new [`database.sync`](https://n6g7.github.io/redux-saga-firebase/reference/database#sync) `event` argument in your code. 🎉

Let me know how it works for you! :D
